docs/monitor-kubernetes.md

```diff
@@ -22,52 +14,94 @@ To set up monitoring of Kubernetes, you need the following:
 
 2. [Helm v3](https://docs.helm.sh/using_helm/#installing-helm).
 3. [kubectl](https://kubernetes.io/docs/tasks/tools/).
+4. The PMM Server API key. The key must have the role "Admin".
 
-## Procedure
-
-### Set up authentication in PMM Server
-
-To access the PMM Server resources and perform actions on the server, configure authentication.
-
-1. Get the PMM API key. The key must have the role "Admin".
+    Get the PMM API key: 
 
-    === "From PMM UI" 
+    === ":material-view-dashboard-variant: From PMM UI" 
 
         [Generate the PMM API key](https://docs.percona.com/percona-monitoring-and-management/details/api.html#api-keys-and-authentication){.md-button} 
 
-    === "From command line"
+    === ":material-console: From command line"
 
         You can query your PMM Server installation for the API
         Key using `curl` and `jq` utilities. Replace `<login>:<password>@<server_host>` placeholders with your real PMM Server login, password, and hostname in the following command:
 
         ``` {.bash data-prompt="$" }
-        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d '{"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d {"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+            ```
+
+        !!! note
+
+            The API key is not rotated. 
+
+## Install the Victoria Metrics Kubernetes monitoring stack
+
```

Comment:
### Quick Install 
Maybe a header can be added

Reply:
Reworked to tabs